src/WinSparkleDotNet/WinSparkleNet.cs

```diff
@@ -0,0 +1,254 @@
+﻿using System;
+using System.Diagnostics;
+using System.Reflection;
+
+namespace WinSparkleDotNet
+{
+    /// <summary>
+    ///     WinSparkle native C# API.
+    /// </summary>
+    public sealed class WinSparkleNet
+    {
+        /// <summary>
+        ///     Gets AssemblyName.Name of assembly.
+        /// </summary>
+        /// <seealso cref="System.Reflection.AssemblyName.Name"/>
+        public string Name { get; private set; }
+        /// <summary>
+        ///     Gets AssemblyCompanyAttribute.Company of assembly.
+        /// </summary>
+        /// <seealso cref="System.Reflection.AssemblyCompanyAttribute"/>
+        public string Company { get; private set; }
+        private string _version;
+        /// <summary>
+        ///     Gets or sets the major, minor, build, and revision numbers of the assembly.
+        /// </summary>
+        /// <seealso cref="System.Reflection.AssemblyName.Version"/>
+        public string Version
+        {
+            get { return _version; }
+            set
+            {
+                Debug.Assert(!_initialized, "Can't set Version after initialization !");
```

Comment:
`initialization !` -> `initialization!`.
Including a space before punctuation is not proper.